kokoro/presubmit/presubmit.sh

```diff
@@ -102,12 +102,21 @@ function run_gofmt() {
   find . -name "*.go" | xargs $GOFMT -w
 }
 
+function run_pylint() {
+  python3 -m pip install pylint --user
+  pylint --rcfile=vulkan_generator/pylintrc vulkan_generator/**/**.py
```

Comment:
Instead of requiring `pylint` being on the `$PATH`, define a `PYLINT` var to provide the path, defaulting to looking for it in `$PATH`, just like all the other linters/checkers used here.